packages/block-editor/src/components/inserter/media-tab/api.js

```diff
@@ -0,0 +1,183 @@
+/* eslint no-console: [ 'error', { allow: [ 'error', 'warn' ] } ] */
+/**
+ * WordPress dependencies
+ */
+import { dispatch, select } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../../store';
+import { unlock } from '../../../lock-unlock';
+
+/**
+ * Interface for inserter media requests.
+ *
+ * @typedef {Object} InserterMediaRequest
```

Comment:
What I need for docs is to export these type definitions(`InserterMediaRequest`, `InserterMediaItem`, `InserterMediaCategoryLabels`, `InserterMediaCategory`). Is there a standarized way to do that, that I'm not aware of?

Currently are docs formatter supports generating only one `typedef` in docs. So one way to go(if there is no standarized way) would be to augment or update our formatter to handle this use case. Another way would be to just add them as comments in the function's description, but I'd like to avoid the duplication of the types descriptions.